backend/src/apiserver/server/run_metric_util.go

```diff
@@ -1,139 +0,0 @@
-// Copyright 2018 The Kubeflow Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package server
-
-import (
-	"regexp"
-
-	"github.com/golang/glog"
-	apiv1beta1 "github.com/kubeflow/pipelines/backend/api/v1beta1/go_client"
-	apiv2beta1 "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client"
-	"github.com/kubeflow/pipelines/backend/src/common/util"
-	"google.golang.org/grpc/codes"
-)
-
-const (
-	// This regex expresses the following constraints:
-	// * Allows lowercase/uppercase letters
-	// * Allows "_", "-" and numbers in the middle
-	// * Additionally, numbers are also allowed at the end
-	// * At most 64 characters
-	metricNamePattern = "^[a-zA-Z]([-_a-zA-Z0-9]{0,62}[a-zA-Z0-9])?$"
-)
-
-// ValidateRunMetricV1 validates RunMetric fields from request.
-func ValidateRunMetricV1(metric *apiv1beta1.RunMetric) error {
```

Comment:
Aren't these still used in v1? or you find no usage to these code?

Reply:
This has been moved to `api_util.go`. It take `model` types instead of API. The goal was to be consistent and obey to the pattern in other services.

Reply:
>(reopen)

Uncommented it. Thank you!